modules/IngestDrugSynonymsSync.py

```diff
@@ -0,0 +1,228 @@
+import requests
+import pandas as pd
+import re
+import json
+import sys
+from pathlib import Path
+import xlrd
+import csv
+import os
+import tempfile
+import numpy as np
+from typing import Optional
+from functools import partial
+import logging
+logger = logging.getLogger('ds')
+
+
+class IngestDrugSynonyms():
+
+    def __init__(self, configPath: Path = Path("config.json")):
+        self.check_config(configPath)
+        self.url_international: str = os.environ["URL_INT"] if isinstance(
+            os.environ["URL_INT"], str) else None
+        self.url_USA: str = os.environ["URL_USA"] if isinstance(
+            os.environ["URL_USA"], str) else None
+        self.url_drugbank: str = os.environ["URL_DRUGBANK"] if isinstance(
+            os.environ["URL_DRUGBANK"], str) else None
+        self.query_keywords: [] = os.environ["QUERY_KEYWORDS"].split(
+            ",") if isinstance(os.environ["QUERY_KEYWORDS"], str) else None
+
+    @staticmethod
+    def check_config(configPath: Path = Path("config.json")):
+        def load_config(configPath: Path):
+            with open(configPath) as f:
+                config = json.load(f)
+                for key in config:
+                    os.environ[key] = config[key]
+        rel_path: Path = Path(os.path.realpath(
+            __file__)).parents[0] / configPath
+        if rel_path.exists:
+            load_config(rel_path)
+        elif configPath.exists:
+            load_config(configPath)
+        else:
+            logger.warning(
+                "Could not load config file from: {}".format(configPath))
+
+    @staticmethod
+    def api(query, from_study, to_study, url):
+        url = url.format(query, from_study, to_study)
+        response = requests.request("GET", url)
+        return response.json()
+
+    def api_wrapper(self, query, from_study):
+        return self.api(query, from_study, from_study+99, self.url_USA)
+
+    def getAllStudiesByQuery(self, query: str) -> list:
+        logger.info("> STARTING scraping with '{}' keyword".format(query))
```

Comment:
When using `logger`, instead of eagerly interpolating, better to use `%s`, so interpolation only executes when in that log leve